testing/symbols/verify_exported.dart

```diff
@@ -28,7 +28,13 @@ void main(List<String> arguments) {
     print('usage: dart verify_exported.dart OUT_DIR [BUILDTOOLS]');
     exit(1);
   }
-  final String outPath = arguments.first;
+  String outPath = arguments.first;
+  if (p.isRelative(outPath)) {
+    /// If path is relative then create a full path starting from the engine checkout
+    /// repository.
+    final String engineCheckoutPath = Platform.environment['ENGINE_CHECKOUT_PATH']!;
```

Comment:
If the environment variable isn't defined, then it might be good to give a more helpful error message.

Reply:
Done! added a few lines to print an error if the env variable doesn't exist.